dev_utils/c4gh.sec.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN CRYPT4GH ENCRYPTED PRIVATE KEY-----
+YzRnaC12MQAGc2NyeXB0ABQAAAAAwAs5mVkXda50vqeYv6tbkQARY2hhY2hhMjBf
+cG9seTEzMDUAPAd46aTuoVWAe+fMGl3VocCKCCWmgFUsFIHejJoWxNwy62c1L/Vc
+R9haQsAPfJMLJSvUXStJ04cyZnDHSw==
+-----END CRYPT4GH ENCRYPTED PRIVATE KEY-----
```

Comment:
This will generate a complaint from GH security about a committed private key.
Add this generation to a script